src/main/kotlin/com/openlattice/mechanic/regenerate/RegenerateIds.kt

```diff
@@ -60,22 +55,18 @@ import java.util.function.Supplier
 private val logger = LoggerFactory.getLogger(RegenerateIds::class.java)
 
 class RegenerateIds(
-        private val pgEdmManager: PostgresEdmManager,
-        private val hds: HikariDataSource,
-        private val ptms: PropertyTypeMapstore,
-        private val etms: EntityTypeMapstore,
-        private val esms: EntitySetMapstore,
+        private val toolbox: Toolbox,
         private val idGen: IdGenerationMapstore,
-        private val principalTrees: PrincipalTreesMapstore,
-        private val executor: ListeningExecutorService
-) {
-    private val entitySets = esms.loadAllKeys()?.map { it to esms.load(it) }?.toMap() ?: mapOf()
-    private val entityTypes = etms.loadAllKeys()?.map { it to etms.load(it) }?.toMap() ?: mapOf()
-    private val propertyTypes = ptms.loadAllKeys()?.map { it to ptms.load(it) }?.toMap() ?: mapOf()
+        private val principalTrees: PrincipalTreesMapstore
+) : Regeneration {
     private val ranges = idGen.loadAllKeys()?.map { it to idGen.load(it) }?.toMap()?.toMutableMap() ?: mutableMapOf()
     private val rangeIndex = AtomicLong()
     private val r = Random()
 
+    override fun regenerate(): Boolean {
+        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
```

Comment:
Looks like this is all unused, presumably b/c this isn't implemented yet

Reply:
Lot of the classes seem to be unused, but I wasn't sure if I should delete it or not. If we all agree, I can just delete it

Reply:
I vote delete as I am a big fan of deleting as much code as possible 👍

Reply:
Looks like 4 votes for delete! 🎉

Reply:
🔪

Reply:
O.O It's fine. I think there is a better way of doing what this was originally intended to do anyway.